src/Sulu/Bundle/AdminBundle/Resources/js/containers/DeleteDependantResourcesDialog/DeleteDependantResourcesDialog.js

```diff
@@ -0,0 +1,261 @@
+        if (this.errored) {
+            return 'error';
+        }
+
+        if (this.cancelled) {
+            return 'warning';
+        }
+
+        return undefined;
+    }
+
+    @computed get snackbarMessage(): string | typeof undefined {
+        const {snackbarType} = this;
+
+        if (!snackbarType) {
+            return undefined;
+        }
+
+        if (snackbarType === 'error' && this.error) {
+            return this.error;
+        }
+
+        if (snackbarType === 'warning') {
+            return translate('sulu_admin.delete_dependants_cancelled_text');
+        }
+
+        return undefined;
+    }
+
+    render() {
+        return (
+            <Dialog
+                cancelText={
+                    this.errored || this.cancelled || this.finished
+                        ? translate('sulu_admin.close')
+                        : translate('sulu_admin.cancel')
+                }
+                confirmDisabled={this.errored || this.cancelled || this.finished}
+                confirmLoading={this.inProgress}
+                confirmText={translate('sulu_admin.delete')}
+                onCancel={
+                    this.errored || this.cancelled || this.finished
+                        ? this.handleClose
+                        : this.handleCancel
+                }
+                onConfirm={this.handleConfirm}
+                onSnackbarCloseClick={this.handleSnackbarCloseClick}
+                open={true}
+                snackbarMessage={this.snackbarMessage}
+                snackbarType={this.snackbarType}
+                title={translate('sulu_admin.delete_dependants_warning_title', {
+                    count: this.dependantResourcesCount,
+                })}
+            >
+                {!this.inProgress && !this.cancelled && !this.finished && !this.errored && (
+                    <p>
+                        {translate('sulu_admin.delete_dependants_warning', {
+                            count: this.dependantResourcesCount,
+                        })}
+                    </p>
+                )}
+
+                {(this.inProgress || this.cancelled || this.finished || this.errored) && (
+                    <React.Fragment>
+                        <div className={styles.progressBar}>
+                            <ProgressBar
+                                max={this.dependantResourcesCount}
+                                type={this.errored
+                                    ? 'error'
+                                    : this.finished
+                                        ? 'success'
+                                        : this.cancelled
+                                            ? 'warning'
+                                            : 'progress'}
+                                value={this.errored && this.totalDeletedResources === 0
+                                    ? 1
+                                    : this.totalDeletedResources
+                                }
```

Comment:
think this would be easier to understand like this:
```suggestion
                                value={this.errored
                                    ? this.totalDeletedResources + 1
                                    : this.totalDeletedResources
                                }
```